docs/common/widget.ldoc

```diff
@@ -39,3 +39,64 @@
 --- Set/get a widget's buttons.
 -- @param _buttons The table of buttons that should bind to the widget.
 -- @function buttons
+
+
+--- When the layout (size) change.
+-- @signal widget::layout_changed
+
+--- When the widget content changed.
+-- @signal widget::redraw_needed
+
+--- When a mouse button is pressed over the widget.
+-- @signal button::press
+-- @tparam table widget The widget
+-- @tparam number lx The horizontal position.
+-- @tparam number ly The vertical position.
```

Comment:
The "l" means "local", I think. This is the position of the mouse press relative to the widget while `geometry` contains the geometry of the widget relative to the wibox (as long as you don't rotate by non-multiple of 90°....).
(Same for button::release)